resources/lang/en/patrols/general/hunting.json

```
@@ -3619,6 +3619,556 @@
                 ]
         }
     ]
+},
+{
+    "patrol_id": "gen_hunt_romanticduo",
+    "biome": ["any"],
+    "season": ["any"],
+    "types": ["hunting"],
+    "tags": ["romantic"],
+    "patrol_art": "cat_running_two",
+    "min_cats": 2,
+    "max_cats": 2,
+    "min_max_status": {
+        "warrior": [2, 2],
+        "apprentice": [-1, -1]
+    },
+    "weight": 30,
+    "intro_text": "p_l and r_c set out to hunt together.",
+    "decline_text": "They are called back to camp to attend to other duties.",
+    "chance_of_success": 50,
+    "success_outcomes": [
+        {
+            "text": "Working as a team, they shut down every possible escape route for their prey and dispatch it easily. p_l compliments r_c's hunting style, and {PRONOUN/r_c/subject} {VERB/r_c/return/returns} the compliment with a gracious smile.",
+            "exp": 20,
+            "weight": 20,
+            "prey": ["medium"],
+            "relationships": [
+{
+            "cats_to": ["p_l"],
+            "cats_from": ["r_c"],
+            "mutual": true,
+            "values": ["romantic", "platonic", "trust", "respect"],
+            "amount": 5
+}
+],
+        "art": "gen_happy_cat"
+},
+        {
+            "text": "p_l is amazed by how intuitively s_c can anticipate {PRONOUN/p_l/poss} movements and how natural it feels to work as a team with {PRONOUN/s_c/object}. After a very successful hunt, p_l and s_c exchange admiring looks, a pleasant tension hanging in the air.",
+            "exp": 20,
+            "weight": 50,
+            "stat_skill": ["SENSE,1", "CLAIRVOYANT,1", "CLEVER,1"],
+            "can_have_stat": ["r_c"],
+            "prey": ["large"],
+            "relationships": [
+{
+            "cats_to": ["r_c"],
+            "cats_from": ["p_l"],
+            "mutual": true,
+            "values": ["romantic", "platonic", "trust", "respect"],
+            "amount": 10
+}
+],
+        "art": "gen_flower_date_warrior_warrior"
+},
+        {
+            "text": "p_l and s_c corner a large prey animal, but just at the last moment, the wind changes and their prey perks up. Without hesitation, s_c bolts forward and slams a paw down on it. p_l is amazed by {PRONOUN/s_c/poss} lightning-fast reflexes and returns to camp a little hot around the ears.",
```

```suggestion
            "text": "p_l and s_c corner a large prey animal, but the wind changes and their prey perks up at the last moment. Without hesitation, s_c bolts forward and slams a paw down on it. p_l is amazed by {PRONOUN/s_c/poss} lightning-fast reflexes and returns to camp a little hot around the ears.",
```
Removing the split here makes it read more naturally and clearly.